java-manta-it/src/test/java/com/joyent/manta/client/MantaClientSnapLinksIT.java

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
The following tests still fail when snaplinks are disabled:

```
MantaClientSnapLinksIT.canMoveDirectoryWithContents
MantaClientSnapLinksIT.canMoveFileToDifferentPrecreatedDirectory
MantaClientSnapLinksIT.canMoveFileToDifferentUncreatedDirectoryCreationEnabled
```

They are not properly skipped.

Reply:
Is there like a flag to check whether snaplinks are disabled for an account ?

Reply:
Noted.